content/en/docs/reference/setup-tools/kubeadm/implementation-details.md

```diff
@@ -207,10 +204,8 @@ Static Pod manifest share a set of common properties:
 
 Please note that:
 
-1. All the images, for the  `--kubernetes-version`/current architecture, will be pulled from `k8s.gcr.io`;
-   In case an alternative image repository or CI image repository is specified this one will be used; In case a specific container image
-   should be used for all control plane components, this one will be used. see [using custom images](/docs/reference/setup-tools/kubeadm/kubeadm-init/#custom-images)
-   for more details
+1. All the images, for the  `--kubernetes-version`, will be pulled from `k8s.gcr.io`;
+   In case an alternative image repository or CI image repository is specified that one will be used; In case a specific container image should be used for all control plane components, that one will be used. See [using custom images](/docs/reference/setup-tools/kubeadm/kubeadm-init/#custom-images) for more details
```

Comment:
i'd replace this with just:

> All the images for this  `--kubernetes-version` will be pulled from `k8s.gcr.io` by default. See [using custom images](/docs/reference/setup-tools/kubeadm/kubeadm-init/#custom-images) for customizing the image repository

Reply:
I re-read this again and maybe we should drop `this --kubernetes-version` from the statement? WDYT? Just say -
`All the images will be pulled from k8s.gcr.io by default.`
Because we don't make any reference to any k8s version before this. So it's not entirely clear what we mean by **this** --kubernetes-version.

Reply:
> All the images will be pulled from k8s.gcr.io by default.

SGTM.

Reply:
Done. Marking this as resolved.